FOUNDATION.md

```diff
@@ -0,0 +1,80 @@
+
+## Values
+The following values were identified in the governance brainstorming exercises during the October 2018 Collaborators Summit in Vancouver. We believe they can operate as guiding principles for Foundation efforts.
+
+### Community Focus
+> TODO: Add succinct definition. Examples from collab summit below.
+
+  - Promote Diversity, Inclusivity and Representation with a global perspective.
+  - Bring members together on a regular basis with equal accessibility
+  - Global Leadership
+  - Be less US / North American centric – engage worldwide
+  - Value moral values
+
+### Project Driven
+> TODO: Add succinct definition. Examples from collab summit below.
+
+  - Projects make all technical decisions
+  - Make projects the priority
+  - Value project input for all decisions
+  - Ensure health of highly used / critical libraries
+  - Help champion new efforts
+  - Provide Recognition
+  - Ensure ability to leave
+
+### Transparent Operation
+> TODO: Add succinct definition. Examples from collab summit below.
+
+  - Open Governance
+  - Radical Collaboration
+  - Value data over opinions
+  - Guarantee the right to disagree
+  - Consensus seeking as a model for decision making
+
+### Simple Structure
+> TODO: Add succinct definition. Examples from collab summit below.
+
+  - By-laws general enough not to require evolution
+  - Simple, project-driven governance evolution
+
+
+## Top Level Committees and Representation
+> TODO: Flesh this out better with the current working model of the two Top Level Committees (CPC and C3) and their representation that was discussed, as is needed to be defined by the *board*'s by-laws *only*. Details on TLC operation, membership, and more detailed responsibilities can be described in their own governance docs.
+- Project representation on the Board
+  - Projects could have tiers, only 'platinum' projects should sit on the Board directly. Other projects would have shared seats (like gold/silver member structure)
+  - Community representation on the Board: 1 seat guaranteed for the Cross Community Council and 1 seat for Individual Membership program if and only if threshold requirements have been met (TODO: detail this more)
+  - Each council (CPC and C3) should have a given allocation of Board seats. How these are allocated should be defined by each respective council.
+- Board must establish Cross Project Council focused on the specific needs of the member projects
+  - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board
+  - CPC should be co-chaired by the Executive Director of the foundation
+
+## Foundation Expectations
```

Comment:
should we land this in your PR or land separately in https://github.com/nodejs/bootstrap/pull/6

similar to other parts I think separation of concerns is a good idea here